packages/sdk/src/reliable_channel/events.ts

```diff
@@ -0,0 +1,69 @@
+import { IDecodedMessage, ProtocolError } from "@waku/interfaces";
+import type { HistoryEntry, MessageId } from "@waku/sds";
+
+export enum ReliableChannelEvent {
+  /**
+   * The message is being sent over the wire.
+   *
+   * This event may be emitted several times if the retry mechanism kicks in.
+   */
+  OutMessageSending = "channel:out:message-sending",
```

Comment:
nit: maybe we can drop `channel:out` and `channel:in` since it is clear from the context that these events are for some channel